File: POO PHP/Aula12/Canguru.php
<?php

require_once 'Mamifero.php';

class Canguru extends Mamifero{

    //Metodo da Classe
    public function usarBolsa() {
        echo "<p>Usando a bolda pra guardar a cria.</p>";
    }
    
    //Metodo de Sobreposição da Classe Mamifero
    public function locomover() {
        echo "<p>Se locomovendo pulando ...</p>";
    }
}

File: POO JAVA/Aula12/src/aula12/Canguru.java
package aula12;

public class Canguru extends Mamifero{

    //Metodo Sobreposto da Classe Mamifero
    @Override
    public void locomover() {
        System.out.println("Pulando ...");
    }
    
    //Metodo da classe
    public void usarBolsa(){
        System.out.println("Usando a bolsa pra guardar sua cria.");
    }
}
